src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -155,12 +155,12 @@ public partial class ImportingConstructorAttribute : System.Attribute
     public partial class ImportManyAttribute : System.Attribute
     {
         public ImportManyAttribute() { }
-        public ImportManyAttribute(string contractName) { }
-        public ImportManyAttribute(string contractName, System.Type contractType) { }
-        public ImportManyAttribute(System.Type contractType) { }
+        public ImportManyAttribute(string? contractName) { }
+        public ImportManyAttribute(string? contractName, System.Type? contractType) { }
```

Comment:
Ditto

Reply:
This one also have doc, but above and below methods documented to allow null